test/unit/runtime/Loader.js

```diff
@@ -230,6 +237,7 @@ suite('Loader.js', function() {
   });
 
   test('LoaderImport.Fail.deperror', function(done) {
+    this.timeout(4000);
```

Comment:
Could this be an issue with our Promise polyfill which uses nextTick on node. If we switch to iojs or node 0.12 we would use v8's Promise instead which has true microtask behavior.

Worth trying at  least.